### PR TITLE
Download all now will use top 1000 instead of top 10000

### PR DESCRIPTION
**Pull request by aprilrd**

### CODE REVIEW

File: src/driver/druidDriver.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ module FacetLegacy {
   }
 
   export class DruidQueryBuilder {
-    static ALL_DATA_CHUNKS = 10000;
+    static ALL_DATA_CHUNKS = 1000;
```
Comment (epistemancer) [inline src/driver/druidDriver.ts]: maybe we should parameterize this?

**Comment by clintropolis** [comment]
should this just be something passed in instead of hard coded?

File: src/driver/druidDriver.ts (hunk 1 above)
Comment (aprilrd) [inline src/driver/druidDriver.ts]: cool... will do that

**Comment by epistemancer** [comment]
:see_no_evil:

**Pull request merged**